packages/api-client/src/views/Request/RequestSection/RequestAuth/RequestAuth.vue

```diff
@@ -58,22 +58,23 @@ const teleportId = `combobox-${nanoid()}`
  * Currently we just filter out empty object for optional but when we add required security we shall handle it!
  */
 const availableSchemes = computed(() => {
+  // TODO: these are commented out for now until we decide how to handle request level requirements
   /** With optional ({}) filtered out */
-  const requestSecurity = activeRequest.value?.security?.filter(
-    (s) => Object.keys(s).length,
-  )
-
-  const base =
-    isReadOnly.value && requestSecurity?.length
-      ? requestSecurity.map((s) => {
-          const nameKey = Object.keys(s)[0]
-          return (
-            Object.values(securitySchemes).find((ss) => ss.nameKey === nameKey)
-              ?.uid ?? ''
-          )
-        })
-      : activeCollection.value?.securitySchemes
+  // const requestSecurity = activeRequest.value?.security?.filter(
```

Comment:
keep this in for now, we will use it for https://github.com/scalar/scalar/issues/3613